logging/bin/deploy_kibana_content.sh

```diff
@@ -122,87 +122,67 @@ if [ "$kibanaready" != "TRUE" ]; then
-         exit 1
-      fi
-   else
-      log_debug "The Kibana tenant space [cluster_admins] exists."
+# Create cluster_admins Kibana tenant space (if it doesn't exist)
+#   Should only be true during UIP scenario b/c our updated securityconfig processing
+#   is bypassed (to prevent clobbering post-deployment changes made via Kibana).
+if ! kibana_tenant_exists "cluster_admins"; then
+   create_kibana_tenant "cluster_admins" "Kibana tenant space for Cluster Administrators"
+   rc=$?
+   if [ "$rc" != "0" ]; then
+      log_error "Problems were encountered while attempting to create tenant space [cluster_admins]."
+      exit 1
    fi
+else
+   log_debug "The Kibana tenant space [cluster_admins] exists."
+fi
 
-   #Migrating from ODFE 1.7.0 to ODFE 1.13.2 (file should only exist during migration)
-   if [ -f "$KB_GLOBAL_EXPORT_FILE" ]; then
-
-      # delete "demo" Kibana tenant space created (but not used) prior to V4m version 1.1.0
-      if kibana_tenant_exists "admin_tenant"; then
-
-         delete_kibana_tenant "admin_tenant"
-
-         rc=$?
-         if [ "$rc" == "0" ]; then
-            log_debug "The Kibana tenant space [admin_tenant] was deleted."
-         else
-            log_debug "Problems were encountered while attempting to delete tenant space [admin_tenant]."
-         fi
-      fi
-
-      log_verbose "Will attempt to migrate Kibana content from previous deployment."
+#Migrating from ODFE 1.7.0 to ODFE 1.13.2 (file should only exist during migration)
+if [ -f "$KB_GLOBAL_EXPORT_FILE" ]; then
 
-      kb_migrate_response="$TMP_DIR/kb_migrate_response.json"
+   # delete 'demo' Kibana tenant space created (but not used) prior to V4m version 1.1.0
+   if kibana_tenant_exists "admin_tenant"; then
 
-      #import previously exported content from global tenant
-      response=$(curl -s -o $kb_migrate_response  -w  "%{http_code}" -XPOST "${kb_api_url}/api/saved_objects/_import?overwrite=false" -H "kbn-xsrf: true"  -H 'securitytenant: cluster_admins'  --form file="@$KB_GLOBAL_EXPORT_FILE"  -u $ES_ADMIN_USER:$ES_ADMIN_PASSWD -k)
+      delete_kibana_tenant "admin_tenant"
 
-      if [[ $response != 2* ]]; then
-         log_warn "There was an issue importing the cached existing Kibana content into the Kibana tenant space [cluster_admins]. [$response]"
-         log_debug "Failed response details: $(tail -n1 $kb_migrate_response)"
-         #TODO: Exit here?  Display messages as shown?  Add BIG MESSAGE about potential loss of content?
+      rc=$?
+      if [ "$rc" == "0" ]; then
+         log_debug "The Kibana tenant space [admin_tenant] was deleted."
       else
-         log_info "Existing Kibana imported to [cluster_admins] Kibana tenant space. [$response]"
-         log_debug "Import details: $(tail -n1 $kb_migrate_response)"
+         log_debug "Problems were encountered while attempting to delete tenant space [admin_tenant]."
```

Comment:
Is this supposed to be a log_debug message or should it be log_warn?

Reply:
It's a DEBUG level message because users are (almost certainly) unaware that this tenant was created as part of the original ODFE 1.7.0 deployment.  Kibana multi-tenancy was disabled so they would have no way of ever bumping into this tenant.  If we cannot delete the tenant, throwing a WARNing is going to make the user think they need to do something.  While we don't really need/want that tenant around, it shouldn't cause problems...and it isn't worth having someone call Tech Support about.